src/core/ext/lb_policy/grpclb/grpclb.c

```diff
@@ -309,6 +309,11 @@ static grpc_lb_policy *create_rr(grpc_exec_ctx *exec_ctx,
     if (parse_ipv4(&uri, &sa, &sa_len)) { /* TODO(dgq): add support for ipv6 */
       memcpy(args.addresses->addrs[out_addrs_idx].addr, &sa, sa_len);
       args.addresses->addrs[out_addrs_idx].len = sa_len;
+      // These are, of course, actually balancer addresses.  However, we
+      // want the round_robin LB policy to treat them as normal backend
+      // addresses, since we don't need to talk to balancers in order to
+      // find the balancers themselves.
+      args.addresses->addrs[out_addrs_idx].is_balancer = false;
```

Comment:
Done.

The double-space-after-period thing is somewhat annoying, since it's a pretty ingrained habit for me (regardless of whether or not it is actually correct :) ).  If this really bothers you, how about throwing together a tool that fixes this automatically, and add a test that it's done correctly, much like we do with clang-format?